test/integration/console_scripts_test.py

```diff
@@ -1,20 +1,18 @@
 """Test the PUDL console scripts from within PyTest."""
 
+import pkg_resources
 import pytest
 
+# Obtain a list of all deployed entry point scripts to test:
+PUDL_SCRIPTS = [
+    ep.name for ep in pkg_resources.iter_entry_points('console_scripts')
+    if ep.module_name.startswith("pudl")
+]
```

Comment:
So we don't have to remember to add every new script to the tests by hand in the future, this grabs a list of all of them from the package resources.